src/service-worker/src/armada/assets.ts

```diff
@@ -23,7 +23,7 @@ class ThrowingFetcher {
 }
 
 export class ArmadaLazyAssetGroup extends LazyAssetGroup {
-  static readonly MAX_ATTEMPTS = 5;
+  static readonly TIMEOUT_MS = 200;
```

Comment:
Are we sure that this timeout is sufficient on testnet?

Reply:
The timeout is just to start making requests to other nodes but the previous requests are still active. So the timeout is like how long you wanna wait before falling back to other nodes. Setting a timeout too low would be doing a race for every fetch because that will overload servers unnecessarily